### DISCUSSION TITLE
有没有可能利用真正的https+网站来绕过gfw？

### DISCUSSION THREAD

**Discussion by Daodanfd5** [v2fly/v2ray-core#1607] [General]
这可能需要极多的服务器端性能消耗，因此应该仅适用于较少人使用的服务器，如果可以实现类似jsproxy的方案的话倒是一个比较好的方案我觉得。

---

例如受密码保护的wordpress，在启用前无法查看内容，是否可以利用这类特征实现在一个正常的网站里进行躲避封锁的检测？比如使用AJAX在网页内浏览其他网页？gfw进行主动探测时只能看见受密码保护的wp文章，应该是很正常的，如果再把它建设成一个普通网站，被墙的概率就更低了。

---

有可能可以基于这类操作实现自动化的翻墙吗？又或许是在tls流量上走udp？

**Comment by Tun2Sucks** [comment]
除非你备案，否则墙才不管你网站是伪装的还是真的

**Comment by Kylejustknows** [comment]
页面代理中转已经存在了20年了。
表面一些的有PHP中转页面转发，底层一些的很多Web服务器也支持各种代理。
难点是隐藏。

现在哪怕你的流量完全包裹在TSL内，只要你用了，整个网站秒封。

**Reply by Daodanfd5** [reply]
我不是这个意思
在wp受密码保护的页面里做代理，由服务器访问网站，然后把网站的内容套在里面
对外伪装成自己的电影站

**Reply by Daodanfd5** [reply]
隐藏用受保护的页面隐藏，然后走iframe框架，访问的是服务器内部代理转换过的网站，人工审查只能看到受保护的页面，实际上也没法找啊，gfw探测整个网站啊全是电影，封的概率更小，甚至还可以找运营商申诉（）

**Reply by Daodanfd5** [reply]
比较像ws+tls+nginx+cdn，但是不太一样的就是伪装更逼真，而且就算墙主动探测，人工和墙也只能看到电影，放点蓝光原碟进去人工判断交换流量很大算是正常的了

浏览器→服务器的真正的页面→服务器内部转换iframe+ajax临时搭建网站→服务器访问要访问的网站→返回数据给客户端

**Reply by Daodanfd5** [reply]
不基于v2ray+websocket，理论上相比于稍微伪装一点的ws+tls+伪装站点是更加逼真的，因为确实只访问了网站的资源

**Reply by Kylejustknows** [reply]
vmess或者ss协议的很多精髓你可能还不知道。

wp加密的页面等技术，都用不上wp，自己写几行php就能搞定。同时现有好几个项目可以让你用“php做中转”看网页。

但无论你如何加密你的“内部页面”，如果你的秘密页面是电影，都没问题。但只要你的秘密页面是中转U管、FB，很快被封。墙就是这么厉害。

如果你在中转时加上混淆（每个html填充废话注释，每个图片填充随机空白，改变大小但不影响你的浏览器渲染），你的网站可能能多撑几分钟，但墙的人工智能算法仍然会从平均熵增量统计数据里，发现你在混淆后看youtube，墙只是多花点时间抓包和分析规律，再封你。

**Comment by Daodanfd5** [comment]
这可不就是隐藏的方法么，wp电影站点，全是电影，封的概率不大

**Comment by ghost** [comment]
V2Ray提供流量混淆，伪装成https来欺骗防火墙。那为啥不直接用真正的https连接V2Ray服务器？

**Comment by happytrudy** [comment]
已经有了overtls
